lib/bb/client.rb

```diff
@@ -51,5 +53,19 @@ def get_download_report(doctype, header_callback, yielder)
       uri = URI.join(config.base_path, "bluebutton/bbreport/#{doctype}")
       streaming_get(uri, token_headers, header_callback, yielder)
     end
+
+    private
+
+    def refresh_final?(attrs)
+      attrs.all? { |e| e[:status].present? }
+    end
+
+    def log_refresh_errors(attrs)
+      failed = attrs.select { |e| e[:status] == 'ERROR' }.map { |e| e[:extract_type] }
+      if failed.present?
+        log_message_to_sentry('Final health record refresh contained one or more error statuses', :warn,
+                              refresh_failures: failed.sort)
```

Comment:
We can iterate on it; for now I think overall number of BB users will be small enough that we can look through sentry and see if any patterns are developing in the next week or so. Statsd counters broken out by extract type would be more suitable long-term. 

Going to merge this so we can get some data sooner rather than later, but I'm entirely open to replacing it later.